rotkehlchen/tests/utils/decoders.py

```diff
@@ -1,9 +1,36 @@
-from unittest.mock import _patch, patch
+from contextlib import ExitStack
+from unittest.mock import patch
 
+from rotkehlchen.chain.ethereum.modules.convex.constants import CPT_CONVEX
+from rotkehlchen.chain.evm.decoding.curve.constants import CPT_CURVE
+from rotkehlchen.chain.evm.decoding.gearbox.constants import CPT_GEARBOX
+from rotkehlchen.chain.evm.decoding.velodrome.constants import CPT_AERODROME, CPT_VELODROME
 
-def patch_decoder_reload_data() -> _patch:
+
+def patch_decoder_reload_data(load_global_caches: list[str] | None = None) -> ExitStack:  # type: ignore[return]  # stack is returned at the end
     """Patch to avoid reloading on-chain data at each decoding"""
-    return patch(
-        target='rotkehlchen.chain.evm.node_inquirer.should_update_protocol_cache',
-        new=lambda *args, **kwargs: False,
-    )
+    with ExitStack() as stack:
+        stack.enter_context(patch(  # patch to not refresh cache by not downloading new data
+            target='rotkehlchen.chain.evm.node_inquirer.should_update_protocol_cache',
+            new=lambda *args, **kwargs: False,
+        ))
+
+        for counterparties, path, patch_general, patch_unique in (  # patch to not load cache from DB by default  # noqa: E501
```

Comment:
What is patch general and patch unique? Can you either add a comment here or add more info to the function's docstring?